config/saturn/warp_user_syms.txt

```diff
@@ -0,0 +1,4 @@
+EntityWarpRoom = 0x060DC040;
```

Comment:
Just FYI, the linker already knows where to put these functions since they're directly compiled into the file. When I fix the splitter to use user defined symbols these would be useful though. 

We have to add user symbols for stuff that's referenced by functions but is extern. So for example if I had a function like:
```
extern bool g_bool;
void example()
{
  g_bool = 1;
}
```

`g_bool` would need a user symbol added.

If these were unknown to the linker, they wouldn't link because they need an underscore in front. So we would eventually want `_EntityWarpRoom` etc.